<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CroppingDialog</class>
 <widget class="QDialog" name="CroppingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>191</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>191</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据裁剪</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始索引：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>结束索引：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_start">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_end">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>120</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_chop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="displayIntegerBase">
    <number>16</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_chop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>裁掉尾部的</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
